server/src/main/java/keywhiz/service/daos/AclDAO.java

```diff
@@ -49,25 +50,20 @@
   private static final Logger logger = LoggerFactory.getLogger(AclDAO.class);
 
   private final DSLContext dslContext;
-  private final ClientDAO clientDAO;
-  private final GroupDAO groupDAO;
-  private final SecretContentDAO secretContentDAO;
-  private final SecretSeriesDAO secretSeriesDAO;
   private final ObjectMapper mapper;
 
   @Inject
-  public AclDAO(DSLContext dslContext, ClientDAO clientDAO, GroupDAO groupDAO,
-      SecretContentDAO secretContentDAO, SecretSeriesDAO secretSeriesDAO, ObjectMapper mapper) {
+  public AclDAO(DSLContext dslContext, ObjectMapper mapper) {
     this.dslContext = dslContext;
-    this.clientDAO = clientDAO;
-    this.groupDAO = groupDAO;
-    this.secretContentDAO = secretContentDAO;
-    this.secretSeriesDAO = secretSeriesDAO;
     this.mapper = mapper;
   }
 
   public void findAndAllowAccess(long secretId, long groupId) {
     dslContext.transaction(configuration -> {
+      DSLContext context = DSL.using(configuration);
```

Comment:
Should give a better name for the DSLContext variables so the wrong one isn't used.